src/Fixer/Basic/BracesFixer.php

```diff
@@ -849,7 +849,7 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
             $previousToken = $tokens[$nextTokenIndex - 1];
             // do not indent inline comments used to comment out unused code
             if (
-                0 === strpos($nextToken->getContent(), '//'.$this->whitespacesConfig->getIndent())
+            (0 === strpos($nextToken->getContent(), '//'.$this->whitespacesConfig->getIndent()) || $nextToken->getContent() === '//')
```

Comment:
Is this indentation change a typo or the result of changes made in the fixer? I think this should be reverted.

Reply:
It was typo, got fixed